stubs/setuptools/pkg_resources/__init__.pyi

```diff
@@ -428,6 +432,7 @@ class Distribution(NullProvider):
     def requires(self, extras: Iterable[str] = ()) -> list[Requirement]: ...
     def activate(self, path: list[str] | None = None, replace: bool = False) -> None: ...
     def egg_name(self) -> str: ...  # type: ignore[override]  # supertype's egg_name is a variable, not a method
+    def __getattr__(self, attr: str) -> Any: ...
```

Comment:
Per our new policy to document any `Any` use: Could you add a comment as to why this is needed? (I think it forwards to another instance.)

Reply:
I think I can copy the docstring directly as a comment, it's clear enough.